Utils/utils.R

```diff
@@ -5,6 +5,18 @@
 # License: MIT License
 #####################################################################################################################
 
+# Init DB using credentials data
+credentials <- data.frame(
```

Comment:
Since we are hardcoding the usernames/passwords, we may have to alert admins (a modal?) that update these credentials the first time the app is loaded.

Reply:
Sure, I can do a modal for this

Reply:
So, I observed the modal:

![image](https://user-images.githubusercontent.com/17878953/128047145-1a6d4907-a65b-47cf-a831-bc3bbed21c11.png)

What are our intentions for these two usernames ("shinyuser", "shinyadmin")? If they are just for demo purposes, I think we keep them hardcoded and don't suggest editing anything?

If they are just for one-time use, perhaps the modal should just keep the "shinyadmin" username hardcoded and lose the shinyuser account. Why? Because if a new user needs to be added, an admin has to do that anyway. However, if someone logs on with "shinyadmin", the modal could be revised to say, "Please open 'administration mode' to create a username with admin rights as "shinyadmin" is only for one time usage." And then when dismissing the modal, it would be awesome if we could somehow highlight the "administration mode" button or re-direct the user there. I realize that may not be (easily) attainable.

![image](https://user-images.githubusercontent.com/17878953/128048018-19dadd0d-2843-417c-9afa-2f7d0e67e662.png)

I'm not convinced the second modal is necessary once you open the "administration mode":
![image](https://user-images.githubusercontent.com/17878953/128049047-aecc1b7d-687b-42ee-9d4f-0de91c31a4dd.png)

Reply:
I like your suggestion @AARON-CLARK of having an admin account and no user account by default.